src/components/veNft/VeNftExtendLockModal.tsx

```diff
@@ -1,30 +1,122 @@
-import { Trans, t } from '@lingui/macro'
-import { Modal } from 'antd'
-import React from 'react'
+import { Form, Modal } from 'antd'
+import { BigNumber } from '@ethersproject/bignumber'
+import { useContext, useEffect, useMemo, useState } from 'react'
+import { ThemeContext } from 'contexts/themeContext'
+import { useExtendLockTx } from 'hooks/veNft/transactor/VeNftExtendLockTx'
+import { NetworkContext } from 'contexts/networkContext'
+import { t, Trans } from '@lingui/macro'
+import { emitSuccessNotification } from 'utils/notifications'
+import { VeNftToken } from 'models/subgraph-entities/v2/venft-token'
 
-interface VeNftExtendLockModalProps {
+import { V2ProjectContext } from 'contexts/v2/projectContext'
+import { useForm } from 'antd/lib/form/Form'
```

Comment:
**suggestion**: We can avoid this import by using `Form.useForm` below instead.

Reply:
May just be worth fixing this globally across the codebase. There are lots of other components using this import.